applets/inhibit/inhibit-applet.c

```diff
@@ -21,9 +21,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#ifdef HAVE_CONFIG_H
```

Comment:
Is it necessary to modify the header file?

Reply:
I think use the follow will be better, This will not modify the header file and source files.
```
add_project_arguments([
  '-DHAVE_CONFIG_H',
  ], language: 'c')
```